superset-frontend/spec/javascripts/sqllab/QueryTable_spec.jsx

```diff
@@ -18,27 +18,33 @@
  */
 import React from 'react';
 import { shallow } from 'enzyme';
-import { Table } from 'reactable-arc';
+import DataTable from '@superset-ui/plugin-chart-table/lib/DataTable';
+import * as useMountedMemo from '@superset-ui/plugin-chart-table/lib/DataTable/utils/useMountedMemo';
 import QueryTable from 'src/SqlLab/components/QueryTable';
 
-import { queries } from './fixtures';
+import { dataTableProps } from 'spec/javascripts/sqllab/fixtures';
 
 describe('QueryTable', () => {
+  // hack for mocking hook that implements sticky behaviour of DataTable
+  jest
```

Comment:
@rusackas This was needed because sticky DataTable uses `useLayoutEffect` under the hood and that caused enzyme not to render the component properly. If you have a better idea how to solve it I'll be happy to try it!